src/ng/animate.js

```diff
@@ -270,10 +270,99 @@ var $AnimateProvider = ['$provide', function($provide) {
     return {
       // we don't call it directly since non-existant arguments may
       // be interpreted as null within the sub enabled function
+
+      /**
+       *
+       * @ngdoc method
+       * @name $animate#on
+       * @kind function
+       * @description Sets up an event listener to fire whenever the animation event (enter, leave, move, etc...)
+       *    has fired on the given element or amount any of its children. Once the listener is fired, the provided callback
```

Comment:
Yes that is a mistake on my end. I too learned English long ago I forget these things :P